### PR TITLE
Update AWS SDK to v2.991.0

### PR DESCRIPTION
**Pull request by sbkg0002**
Should fix https://github.com/alexjurkiewicz/ecr-scan-image/issues/16

### CODE REVIEW

File: package-lock.json [hunk 1]
```diff
@@ -31,8 +31,8 @@
       }
     },
     "aws-sdk": {
-      "version": "2.620.0",
-      "resolved": "https://registry.npmjs.org/aws-sdk/-/aws-sdk-2.620.0.tgz",
+      "version": "2.991.0",
+      "resolved": "https://registry.npmjs.org/aws-sdk/-/aws-sdk-2.991.0.tgz",
       "integrity": "sha512-j+PGKbYwk5OjzbCFbxzRkcJVEvfdn9YFkrU8dJdjleYf9AzUSUpTjMyZsyUPOKxJxdNdA9FYTs43WdtSKHZa0A==",
```
Comment (alexjurkiewicz) [inline package-lock.json:36]: how come the hash wasn't updated? Did you test this with `npm i`?

**Review by alexjurkiewicz — CHANGES_REQUESTED** [review]
Thanks for the PR! Comments below

File: package-lock.json (hunk 1 above)
Comment (sbkg0002) [reply package-lock.json]: Sorry @alexjurkiewicz, my bad; I was just lazy. It is updated, `npm` just installs it.

**Review by alexjurkiewicz — APPROVED** [review]

**Comment by alexjurkiewicz** [comment]
thank you!

**Pull request merged**